_posts/2020-02-05-keycloak-z-angularem-i-keycloakiem.md

```diff
@@ -377,7 +377,7 @@ Jednak jeśli będąc zalogowanym użytkownikiem, wylogujemy się z backendu a n
 
 Dlaczego?
 
-O tym jak działa Keycloak i tokeny którymi się posługujemy (a dokładniej protokół OpenID) nie jest tematem tego wpisu. Co do samego wylogowywania, to jest to temat na osobny wpis, ponieważ nie jest wcale taki prosty do zaimplementowania, szczególnie jeśli mówimy o rozproszonych systemach z wieloma instancjami aplikacji stojącymi za loadbalancerem.
+O tym jak działa Keycloak i tokeny którymi się posługujemy (a dokładniej standard OpenID Connect) nie jest tematem tego wpisu. Co do kwestii wylogowywania, to temat (ze względu na złożoność implementacji) nadaje się na osobny wpis, szczególnie gdy mówimy o rozproszonych systemach z wieloma instancjami aplikacji, stojącymi za loadbalancerem.
```

Comment:
przecinek:
O tym jak działa Keycloak i tokeny, którymi się posługujemy (a dokładniej standard OpenID Connect) nie jest tematem tego wpisu. Co do kwestii wylogowywania, to temat (ze względu na złożoność implementacji) nadaje się na osobny wpis, szczególnie gdy mówimy o rozproszonych systemach z wieloma instancjami aplikacji, stojącymi za loadbalancerem.